src/Maestro/ReleasePipelineRunner/ReleasePipelineRunner.cs

```diff
@@ -282,14 +284,22 @@ public async Task ProcessFinishedReleasesAsync(CancellationToken cancellationTok
                         }
                         else
                         {
-                            Logger.LogInformation($"All releases for build {buildId} for channel {channelId} finished. Creating BuildChannel.");
+                            if (successfulRelease)
```

Comment:
I think if a build has two releases pipeline, one is in progress and the other failed this code won't be executed and we'll keep creating GH issues until all releases for the build are finished.